openshift/console
# OCPBUGS-17203: mock apis for git repo in test serverless function tests

**Fix**: https://issues.redhat.com/browse/OCPBUGS-17203
**Problem**: Running test serverless function tests were giving rate limit issue

**Solution**: Mock the git url to avoid the rate limit issue.

**Test Setup:**
- Cluster should be running on local
- Change TAGS in frontend/packages/knative-plugin/integration-tests/cypress.json file to : "@smoke or @regression and not (@manual or @to-do or @un-verified)", 

- Command to execute:
     - yarn run dev
     -  yarn run test-cypress-knative
 

- Execute the serverless-function.feature file

**Browser**
Chrome 114

**Test Execution Screenshot:**
